wagtail/admin/widgets/auto_height_text.py

```diff
@@ -5,7 +5,7 @@
 
 
 class AdminAutoHeightTextInput(widgets.Textarea):
-    template_name = "wagtailadmin/widgets/auto_height_text_input.html"
+    attrs["data-controller"]="w-auto-size"
```

Comment:
did you manually test this? I am pretty sure this will not work and throw an error. You will likely need to do this in the `__init__`